headers/functions/overlay29.h

```diff
@@ -227,12 +228,18 @@ int GetMovePower(struct entity* entity, struct move* move);
 bool UpdateStateFlags(struct monster* monster, uint16_t mask, bool set_flags);
 void AddExpSpecial(struct entity* attacker, struct entity* defender, int base_exp);
 void EnemyEvolution(struct entity* enemy);
+bool LevelUpItemEffect(struct entity* attacker, struct entity* defender, int levels, bool message,
+                       bool dialog);
 bool TryDecreaseLevel(struct entity* user, struct entity* target, int n_levels);
-bool LevelUp(struct entity* user, struct entity* target, bool message, undefined4 param_4);
+bool LevelUp(struct entity* user, struct entity* target, bool message, bool dialog);
 void EvolveMonster(struct entity* monster, undefined4* param_2, enum monster_id new_monster_id);
```

Comment:
Since you swapped params 1 and 2 in the symbol tables, you should swap them here too

Reply:
I think you meant for EvolveMonster. After double checking my original work, I will be correcting this and making a change to EnemyEvolution as well. (I was double checking I meant to swap param 1 and 2).